### DISCUSSION TITLE
Channels with Alternative Langauges

### DISCUSSION THREAD

**Discussion by jkemp814** [iptv-org/community#226] [Q&A] [answered]
Now that the playlist tags are becoming more synchronized, it's getting easier to discern the duplicates in the list. There is now an issue of why am I still seeing duplicates with some channels?

They have alternative languages.

Take for example: Pluto TV MTV. Even thou there are about eight of them, three still show in the list. But there is currently no way to tell by the description. 

Should there be: (ESP) for Spanish, and something for French, Chinese, Brazil, and Italy, etc...

That way, the viewer knows which one they prefer to click on. Right now, many of them have the exact same name, causing one to think they are duplicates when they are not.

**Comment by Dum4G** [comment, marked as answer]
You can add postfixes like (Spanish subtittles) or call them MTV Spain for ex

**Reply by BellezaEmporium** [reply]
I agree on that idea. Just add a little something that shows in which language the channel is in.
